physics/ksp_system_test.cpp

```diff
@@ -0,0 +1,469 @@
+        std::move(barycentric_system.degrees_of_freedom),
+        ksp_epoch,
+        integrators::McLachlanAtela1992Order5Optimal<Position<KSP>>(),
+        45 * Minute,
+        1 * Milli(Metre));
+  }
+
+  Instant const ksp_epoch;
+
+  KSPCelestial sun;
+  KSPCelestial eeloo;
+  KSPCelestial jool;
+  KSPCelestial pol;
+  KSPCelestial bop;
+  KSPCelestial tylo;
+  KSPCelestial vall;
+  KSPCelestial laythe;
+  KSPCelestial dres;
+  KSPCelestial duna;
+  KSPCelestial ike;
+  KSPCelestial kerbin;
+  KSPCelestial minmus;
+  KSPCelestial mun;
+  KSPCelestial eve;
+  KSPCelestial gilly;
+  KSPCelestial moho;
+  std::vector<not_null<KSPCelestial*>> const all_bodies = {&sun,
+                                                           &eeloo,
+                                                           &jool,
+                                                           &pol,
+                                                           &bop,
+                                                           &tylo,
+                                                           &vall,
+                                                           &laythe,
+                                                           &dres,
+                                                           &duna,
+                                                           &ike,
+                                                           &kerbin,
+                                                           &minmus,
+                                                           &mun,
+                                                           &eve,
+                                                           &gilly,
+                                                           &moho};
+  std::vector<not_null<KSPCelestial*>> const planets_and_moons = {&eeloo,
+                                                                  &jool,
+                                                                  &pol,
+                                                                  &bop,
+                                                                  &tylo,
+                                                                  &vall,
+                                                                  &laythe,
+                                                                  &dres,
+                                                                  &duna,
+                                                                  &ike,
+                                                                  &kerbin,
+                                                                  &minmus,
+                                                                  &mun,
+                                                                  &eve,
+                                                                  &gilly,
+                                                                  &moho};
+};
+
+TEST_F(KSPSystemTest, KerbalSystem) {
```

Comment:
This test has no expectations whatsoever.  Could we add a few to make sure that things make sense (eg, the bodies remain reasonably close to one another).  Doesn't need to be very tight, but it should fail if we have a major regression.